src/main/java/com/ust/sourcecourse/configuration/controller/ColumnsController.java

```diff
@@ -0,0 +1,36 @@
+package com.ust.sourcecourse.configuration.controller;
+
+import java.net.URI;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+import org.springframework.web.util.UriComponents;
+
+import com.ust.sourcecourse.configuration.entity.ColumnsEntity;
+import com.ust.sourcecourse.configuration.entity.SourcePocEntity;
+import com.ust.sourcecourse.configuration.repository.SourcePocRepository;
+import com.ust.sourcecourse.configuration.service.ColumnsService;
+
+public class ColumnsController {
+	@Autowired
+	private ColumnsService columnsService;
+	@GetMapping("/getcolumsdetails")
```

Comment:
Have proper URL mappings. Instead of _getcolumsdetails_, You can add a common URL in controller level and remove this URL